src/lib/ScannerCommand.ts

```diff
@@ -31,6 +27,11 @@ export abstract class ScannerCommand extends SfdxCommand {
 			filters.push(new RuleFilter(FilterType.RULENAME, [this.flags.rulename]));
 		}
 
+		// TODO: This isn't idiomatic. https://github.com/rwaldron/idiomatic.js
```

Comment:
This TODO could be more helpful. The link is great, but ideally, the comment itself should also should explain what idiomatic means in this context, why this code isn't idiomatic, and how we could make it idiomatic.
Also, are the `if` branches at lines 10, 15, 20, and 26 also not idiomatic?

Reply:
Good point. I didn't want to change a bunch of code written by others, but i also didn't want to make this line different. 

The conventional check would be `if (this.flags.engine && this.flags.engine.length) {` 

Following the most common conventions for a given language cuts down on errors and cognitive load of those new to the code.

Any objection to updating all of the checks to this pattern? CC: @rmohan20